include/jwt-cpp/jwt.h

```diff
@@ -2668,6 +2668,162 @@ namespace jwt {
 			}
 		}
 	};
+	
+	template<typename json_traits>
+	class jwks {
```

Comment:
This seems to more closely match a `JWK` from https://tools.ietf.org/html/rfc7517 than the `JWK Set` from https://tools.ietf.org/html/rfc7517#section-5

I think the naming here and for `jwks_keys` are a little confusing for me.

Perhaps this could be called a `jwk` since it's one single key descriptor, and the following would be a `jwks`